frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -475,127 +478,116 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                     </Form>
                 ) : (
                     <>
-                        {featureFlagLoading ? (
```

Comment:
This is never hit because of https://github.com/PostHog/posthog/pull/20042/files?diff=unified&w=1#diff-ed6684325dbfb464b04e4cd80f6245863b2ca34d4f77befae4f36c6b74be6c9bR124